uktrade/data-hub-frontend
# Sector react typeahead

## Description of change

Adds a typeahead to the investment project react view to allow searching by Sector.

_This will need further refactoring when other typeaheads using the metadata are added - I didn't want to make it too generic before it was clear what would be needed._

## Test instructions

Run locally; search for a project sector (currently hardcoded to the values in the fixture database - will make a new ticket to update the BE fixtures to use the updated ids before adding the new sector id list). The number of projects should change, and a chip appear with the filter name. If you copy and paste the URL to a new browser tab, the filters should still be visible and working.

## Screenshots
<img width="1036" alt="Screenshot 2020-11-19 at 11 06 17" src="https://user-images.githubusercontent.com/23265724/99658361-4d293680-2a57-11eb-85ee-06060e44129b.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to ~master~ feature-merge branch?
